.github/workflows/config/yamllint_config.yml

```diff
@@ -34,6 +39,5 @@ rules:
   new-lines: enable
   octal-values: disable
   quoted-strings: disable
-  trailing-spaces: enable
-  truthy:
-    level: warning
+  trailing-spaces: disable
```

Comment:
The yaml generator sometimes leaves trailing spaces. 

I have re-enabled the trailing spaces rule and edited all the generated code to prevent the warning. 

I have also cleaned up yaml formatting causing three other warnings, so we can have 100% clean yamllint output unless a new problem is introduced, which will make the output more useful:

1. comment not indented like content
2. missing starting space in comment
3. missing document start "---"